comms/src/control_service/mod.rs

```diff
@@ -52,6 +52,7 @@
 //! let conn_manager = Arc::new(ConnectionManager::new(context.clone(), node_identity.clone(), peer_manager.clone(), PeerConnectionConfig {
 //!      max_message_size: 1024,
 //!      max_connect_retries: 1,
+//!      max_connections: 10,
```

Comment:
```suggestion
//!      max_connections: 100,
```
Should this not reflect the default?